cmd/cluster/access/access.go

```diff
@@ -49,23 +49,22 @@ var (
 )
 
 // NewCmdCluster implements the 'cluster access' subcommand
-func NewCmdAccess(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+func NewCmdAccess(streams genericclioptions.IOStreams, client *k8s.LazyClient) *cobra.Command {
+	ops := newClusterAccessOptions(client, streams)
 	accessCmd := &cobra.Command{
-		Use:               "break-glass <cluster identifier>",
+		Use:               "break-glass <cluster identifier> --reason <OHSS or PD ticket>",
```

Comment:
Flags shouldn't be included in a command's `Use` field

Reply:
removed